_maps/map_files/SeleneStation/SeleneStation.dmm

```diff
@@ -17185,11 +17185,10 @@
 /turf/open/floor/iron,
 /area/station/hallway/primary/fore)
 "fKc" = (
-/obj/structure/closet/crate/grave,
-/obj/item/circuitboard/machine/circulator,
-/obj/item/circuitboard/machine/generator,
-/obj/item/circuitboard/machine/circulator,
-/turf/open/floor/grass,
```

Comment:
NO

Reply:
It's the grave subtype that kills you, if that helps